javascript/680-Valid-Palindrome-II.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Linear Time.
+ * Time O(N) | Space O(1)
+ * https://leetcode.com/problems/valid-palindrome-ii
+ * @param {string} s
+ * @return {boolean}
+ */
+//
+var validPalindrome = function(s) {
+
+    let left = 0;
+    let right = s.length - 1;
+
+    while (left < right) {
+        if (s[left] !== s[right]) {
+            const isValid = isPalindrome(s, left, right) || isPalindrome(s, left + 1, right + 1);
+            if (isValid) return true;
```

Comment:
You can remove if else and just return isValid

Reply:
That is a great catch!! I'm so stupid. That's great, why didn't I think of that? Thanks, @aakhtar3. I updated the code please check.